compiler/src/dotty/tools/dotc/ast/DesugarEnums.scala

```diff
@@ -322,6 +322,12 @@ object DesugarEnums {
       (ordinal, Nil)
   }
 
+  def enumBaseMeths(using Context): List[Tree] =
+    if isJavaEnum then
+      enumLabelMeth(EmptyTree) :: Nil
+    else
+      ordinalMeth(EmptyTree) :: enumLabelMeth(EmptyTree) :: Nil
```

Comment:
What was the reason for adding `enumLabel` again? Why not just use `toString`?

Reply:
This case has been modified to generate either `Nil` (if parent is `jl.Enum`) or `ordinalMeth(EmptyTree) :: Nil`

Reply:
this is no longer part of the implementation